src/cmake/compiler.cmake

```diff
@@ -5,7 +5,7 @@ message (STATUS "CMAKE_CXX_COMPILER is ${CMAKE_CXX_COMPILER}")
 message (STATUS "CMAKE_CXX_COMPILER_ID is ${CMAKE_CXX_COMPILER_ID}")
 
 set (USE_CPP 11 CACHE STRING "C++ standard to prefer (11, 14, 17, etc.)")
-option (USE_LIBCPLUSPLUS "Compile with clang libc++")
+option (USE_LIBCPLUSPLUS "Compile with clang libc++" OFF)
```

Comment:
Off by default here, but set to "1" in the helptext above.

Reply:
You mean the helptext in the Makefile? You may consider this blasphemy, but those tend to show how to change the defaults, they don't give examples of redundantly setting the defaults. I know that may be backwards, but if we don't like it, we should change it in a separate pass.